src/routes/github/repository/github-repository-get.ts

```diff
@@ -79,10 +79,16 @@ const getReposBySubscriptions = async (repoName: string, subscriptions: Subscrip
 					return [];
 				});
 
-			return installationSearch;
+			// We don't want to return repos that are not in Database, otherwise we won't be able to fetch branches to branch from later
+			// The app can be installed in a GitHub org but that org might not be connected to Jira, therefore we must filter them out, or
+			// the next steps (e.g. get repo branches to branch of) will fail
+			const subscriptionRepos = await RepoSyncState.findAllFromSubscription(subscription);
+			const subscriptionRepoIds = new Set(subscriptionRepos.map(repo => repo.repoId));
+			return installationSearch.filter(repo => subscriptionRepoIds.has(repo.id));
```

Comment:
/nit You could probably get away without the `new Set`... but not certain how the data is structured so might not 😆 

```
const subscriptionRepoIds = subscriptionRepos.map(repo => repo.repoId);
return installationSearch.filter(repo => repoIds.includes(repo.id));
```